ci/docker-compose.yml

```diff
@@ -89,7 +89,7 @@ services:
       KUDU_MASTER: "false"
 
   omniscidb:
-    image: omnisci/core-os-cpu:v5.1.0
+    image: omnisci/core-os-cpu-dev:latest
```

Comment:
@semelianova does it really need change omniscidb image here?
if yes, we need to wait for #2084